_layouts/home.html

```diff
@@ -136,11 +136,13 @@ <h3>
     {% endfor %}
   </div>
 
+  {% if visible_posts.size > 12 %}
```

Comment:
```suggestion
  {% if visible_posts.size > 5 %}
```
@MisterRios I agree 12 can be too many, I think 5 is a good limit, what do you think ?

Reply:
The posts are grouped in groups of 3, so I think 6 is probably the sweet spot. Would that also be okay?